### ISSUE TITLE
error when only getter is defined for property

### ISSUE THREAD

**Issue by tvdr** [ryangjchandler/spruce#151]
Rollup enables strict mode by default, which then throws TypeError when no setter is defined for an object property getter (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Getter_only)

**Comment by ryangjchandler** [comment]
@tvdr Would you be able to provide any more information on your problem?

**Comment by tvdr** [comment]
Sure, thanks for the quick reply.

when using the below store obejct  

```
Spruce.store('csengetesirend', {
    show: true,
    orak: [
        {id: 1, tol: '8:00', ig: '8:45', szunet: 10},
        {id: 2, tol: '8:55', ig: '9:40', szunet: 10},
        {id: 3, tol: '9:50', ig: '10:35', szunet: 15},
        {id: 4, tol: '10:50', ig: '11:35', szunet: 5},
        {id: 5, tol: '11:40', ig: '12:25', szunet: 20},
        {id: 6, tol: '12:45', ig: '13:30', szunet: 5},
        {id: 7, tol: '13:35', ig: '14:20', szunet: 5},
        {id: 8, tol: '14:25', ig: '15:10', szunet: 5},
        {id: 9, tol: '15:15', ig: '16:00', szunet: false},
    ],
    get orakforcalendar() {
        var orak = this.orak.map(function (item) {
            return {
                id: 'csengetes_' + item.id,
                title: item.id + '.óra',
                startTime: item.tol,
                endTime: item.ig,
                display: 'background'
            }
        })

        return orak;
    },
    showinfulcalendar() {

        var thisstore = this;

        this.orakforcalendar.map(function (item) {
            var oldevent = f_calendar.getEventById(item.id);

            if (oldevent != null) {
                oldevent.remove();
            }


            if (thisstore.show) {
                f_calendar.addEvent(item);
            }

        })
    }
})
```
(sorry for the hungarian variable names)

observable.js throws "Uncaught TypeError: Cannot set property orakforcalendar of #<Object> which has only a getter"

While debugging, i found that it works until version 2.6.4, when you migrated to build with rollup.

Rollup emits `'use strict';`  by default in the dist file, which, accorording to the article on the above linked Moz Dev page, throws error when no setter method is defined to an object property getter.

**Comment by ryangjchandler** [comment]
Okay, this can be fixed by setting `strict: false` on the output.

**Comment by ryangjchandler** [comment]
Resolved in v2.7.1.
